javascript-functions/main.js

```diff
@@ -0,0 +1,33 @@
+function convertMinutesToSeconds(minutes) {
+  var seconds = minutes * 60;
+  console.log(minutes + ' minutes is = to ' + seconds + ' seconds.');
```

Comment:
so instead of using console logs here, you want to return the value, the call the function, save the return value in a variable, and then console log that variable as in the gif example, let me know if you have any questions on it!